docs/source/example_notebooks/functional_api.ipynb

```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dowhy as dw\n",
+    "from dowhy.causal_identifier import (\n",
+    "    AutoIdentifier,\n",
+    "    BackdoorAdjustment,\n",
+    "    IDIdentifier,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
```

Comment:
Some markdown suggestions: 

I imagine that we can continue to update the same notebook as the new API for different functions (e.g., graph, estimate, refute) is implemented. So in that case, here are some things that will be good to add:
1. A title for this notebook. "Functional API preview"
2. An introductory text that provides context. "This notebook provides a preview of the proposed functional API for dowhy. For details on the new API for DoWhy, check out https://github.com/py-why/dowhy/wiki/API-proposal-for-v1 It is a work-in-progress and is updated as we add new functionality. We welcome your feedback through Discord or on the  [Discussions page](https://github.com/py-why/dowhy/discussions/429).

This functional API is designed with backwards compatibility. So both the old and new API will continue to co-exist and work for the immediate new releases. Gradually the old API using CausalModel will be deprecated in favor of the new API."
something like this will be good to add as an intro. Feel free to edit/add as you see fit
 
"